bin/katello-certs-check

```diff
@@ -124,6 +124,30 @@ function check-non-ascii () {
     fi
 }
 
+
+function check-server-cert-encoding () {
+    printf "Checking server certificate's encoding: "
+    openssl x509 -inform pem -in $CERT_FILE -noout -text &> /dev/null
+    if [ $? == "0" ]; then
+        success
+    else
+        openssl x509 -inform der -in $CERT_FILE -noout -text &> /dev/null
+        if [ $? == "0" ]; then
+            printf "[FAIL]\n"
```

Comment:
Thank you. I will amend the error messages so that they are sent to `stderr`.

Until that is done, this PR should be considered a Work In Progress (WIP) and **NOT** merged.